packages/bridge-ui/src/components/BridgeForm/SelectToken.svelte

```diff
@@ -33,6 +37,17 @@
     closeDropdown();
   }
 
+  function showAddERC20() {
+    if (!$signer) {
+      // No need to waste user's time showing them something
+      // they can't use
+      warningToast('Please, connect your wallet.');
```

Comment:
not sure this comma is necessary/granmatiicaly correct/typical?

Reply:
Ok, let me get rid of it